.github/workflows/release-amd64.yaml

```diff
@@ -10,6 +10,7 @@ jobs:
   build-kata-static-tarball-amd64:
     uses: ./.github/workflows/build-kata-static-tarball-amd64.yaml
     with:
+      push-to-registry: yes
```

Comment:
Hi @stevenhorsman !

In `./.github/workflows/build-kata-static-tarball-amd64.yaml` with `push-to-registry=yes` it will log-in quay.io but push the image to ghcr.io.  Unless I missed something, we should log-in the right registry :D (applies to the arm64, ppc64le and s390x workflows).

Reply:
hmm... just realized that it's passing  ARTEFACT_REGISTRY_USERNAME and  ARTEFACT_REGISTRY_PASSWORD variables. So likely internally, somewhere, it's loging to ghcr.io

Reply:
Yeah - I was puzzled by the ghcr.io login, but assumed it must been done auto-magically by github

Reply:
Found it: https://github.com/kata-containers/kata-containers/blob/main/tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh#L1092